.changeset/breezy-pots-sort.md

```diff
@@ -0,0 +1,4 @@
+---
+---
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏add patch for `cli` package

This PR is mostly about tests, but it does change logic that could affect behavior. We should give it a proper version.